Started by user [8mha:////4Gm8z8svCaXKqFzr1Z5zntsQFmxEgC1XfjqScehTkFs8AAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/dLi1CL9xJTczDwAj6GcLcAAAAA=[0mnazar
Running as SYSTEM
Building remotely on [8mha:////4Gb5jqnYN5atl4H6TvVp5YbnrSRqULYVnZcIQhut4W5hAAAAnx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzOEgZh/eT83ILSktQi/TBfI93inMSyVAA+LWVRyAAAAA==[0mVM2-slave (linux_vm2_slave) in workspace /home/user1/workspace/ansible-docker
[ansible-docker] $ /bin/sh -xe /tmp/jenkins5741427580089147230.sh
+ echo 172.31.38.161 ip-172-31-38-161.us-east-2.compute.internal 172.31.32.48 ip-172-31-32-48.us-east-2.compute.internal
172.31.38.161 ip-172-31-38-161.us-east-2.compute.internal 172.31.32.48 ip-172-31-32-48.us-east-2.compute.internal
+ sudo chmod 666 /etc/ansible/hosts
+ sudo sed -i '/ansibleslave\]/d' /etc/ansible/hosts
+ sudo sed -i /172.31.32.48/d /etc/ansible/hosts
+ sudo echo '[ansibleslave]'
+ sudo echo 172.31.32.48
[ansible-docker] $ /bin/sh -xe /tmp/jenkins3490568728810637009.sh
+ sudo touch /etc/ansible/playbook-docker-uninstall.yml
+ sudo chmod 755 /etc/ansible/playbook-docker-uninstall.yml
+ sudo tee /etc/ansible/playbook-docker-uninstall.yml
[ansible-docker] $ /bin/sh -xe /tmp/jenkins2958341209557116091.sh
+ sudo touch /etc/ansible/playbook-docker.yml
+ sudo chmod 755 /etc/ansible/playbook-docker.yml
+ sudo tee /etc/ansible/playbook-docker.yml
+ echo 42
42
+ echo 42
42
[ansible-docker] $ /bin/sh -xe /tmp/jenkins1370755159923500972.sh
++ ls /home/user1/42/
ls: cannot access /home/user1/42/: No such file or directory
+ projectname=
Build step 'Execute shell' marked build as failure
Finished: FAILURE
